site/.eslintrc.yaml

```diff
@@ -32,6 +32,16 @@ plugins:
   - react-hooks
   - jest
   - unicorn
+  - testing-library
+overrides:
+  - files: ["**/__tests__/**/*.[jt]s?(x)", "**/?(*.)+(spec|test).[jt]s?(x)"]
+    extends: ["plugin:testing-library/react", "plugin:testing-library/dom"]
+    rules:
+      # These rules are disabled because they can degrade test speed a bit since
+      # sometimes we need to access the directly in a lower scope to prevent the
+      # library to fetch a bunch of element
```

Comment:
Sorry, I don't understand this comment. Could you try to explain a little differently?

Reply:
Related to this: https://coder.com/docs/v2/latest/contributing/frontend#tests-getting-too-slow

Reply:
Ah, gotcha. Maybe we could link those docs and phrase it slightly differently:
"Occasionally, we must traverse the DOM when querying for an element to avoid the performance costs that come with using selectors like `ByRole`. You can read more about these performance costs [here]( https://coder.com/docs/v2/latest/contributing/frontend#tests-getting-too-slow)."

Reply:
Loved, ty!